src/components/FileHistory/HistoryRow.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+import { Media, Bd, Img } from 'cozy-ui/transpiled/react/Media'
+import { Caption, SubTitle, Bold } from 'cozy-ui/transpiled/react/Text'
+import { Icon, Circle } from 'cozy-ui/transpiled/react'
+import palette from 'cozy-ui/react/palette'
+
+import styles from './styles.styl'
+
+const denseStyle = { height: '48px' }
+
+//@todo this component will be moved to UI when polished with an HistoryTimeLine
```

Comment:
Since this is (I assume) heavily based on `CompositeRow`, could you add a comment about the main differences?